integration_tests/test_asset_group_sightings.py

```diff
@@ -138,7 +138,7 @@ def test_asset_group_sightings(session, login, codex_url, test_root):
                 'updated': assets[0]['updated'],
             },
         ],
-        'comments': 'None',
```

Comment:
Does this mean EDM has been changed so that tests will fail today?

Reply:
(just following up, post-slack conversation.)  nope, this change was all within houston in this pr.